verify.sh

```diff
@@ -241,3 +241,42 @@ else
 fi
 echo -e "${dim} - Version: ${subbold}$VER"
 echo -e ""
+
+# ecowitt
```

Comment:
I love this!  One request:  Can we put this whole ecowitt test within a conditional to see if the config file is set up?  If the config file is missing, skip the test.  Thoughts?